internal/framework/flex/autoflex.go

```diff
@@ -550,7 +556,23 @@ func (expander autoExpander) mapOfString(ctx context.Context, vFrom basetypes.Ma
 
 					vTo.Set(reflect.ValueOf(to))
 					return diags
+				default:
+					diags.Append(vFrom.ElementsAs(ctx, &vTo, true)...)
+					if diags.HasError() {
+						return diags
+					}
+
+					vTo.Set(reflect.ValueOf(vTo))
+					return diags
 				}
+			default:
+				diags.Append(vFrom.ElementsAs(ctx, &tMapElem, false)...)
```

Comment:
Is there a useful comment that could be added here?

Reply:
That was part of debugging and shouldn't be there. Thanks for catching!